docs/sphinx/source/contributing.rst

```diff
@@ -162,20 +196,39 @@ pvlib python generally follows the `PEP 8 -- Style Guide for Python Code
 <https://www.python.org/dev/peps/pep-0008/>`_. Maximum line length for code
 is 79 characters.
 
-Documentation must be written in `numpydoc format <https://numpydoc.readthedocs.io/>`_.
+Code must be compatible with python 2.7 and 3.4+.
```

Comment:
Not for this PR, but do we have a plan/statement about when we will stop 2.7 support?

Reply:
@thunderfish24 https://github.com/pvlib/pvlib-python/issues/501